src/objective/rank_objective.hpp

```diff
@@ -104,6 +105,14 @@ class LambdarankNDCG: public ObjectiveFunction {
     }
     std::stable_sort(sorted_idx.begin(), sorted_idx.end(),
                      [score](data_size_t a, data_size_t b) { return score[a] > score[b]; });
+    // get best and worst score	
+    const double best_score = score[sorted_idx[0]];
+    data_size_t wrost_idx = cnt - 1;
```

Comment:
Mis-correction in b84c14e2ccbf5d40c793e73625c1ebd1c232a36a: now two variables have `wrost_` prefix instead of `worst`.